api/clients/v2/examples/client_construction.go

```diff
@@ -34,8 +34,8 @@ const (
 	registryCoordinatorAddress = "0x53012C69A189cfA2D9d29eb6F19B32e0A2EA3490"
 	// These two addresses are no longer required for the Eth Client, but parameter is still being taken until we deprecate the flags
 	eigenDAServiceManagerAddress = ""
-	// blsOperatorStateRetrieverAddress is still used for CertBuilder
-	blsOperatorStateRetrieverAddress = "0x003497Dd77E5B73C40e8aCbB562C8bb0410320E7"
+	// operatorStateRetrieverAddress is still used for CertBuilder
```

Comment:
iiuc the core protocol supports quorum signature schemas for both BLS and ECDSA which I think is why this prefix was used?

I only partially know what I'm talking about here but if there's such a thing as a ECDSAOperatorStateRetriever and if for whatever reason we see that being added to our offchain logic then having this existing specificity might be good for long-term evolvability (very unlikely though)

Reply:
Yes, asked protocol guild on slack.
But basically I don't see why we would ever use an ECDSAOperatorStateRetriever, but if we ever do we can add the prefix.
IMO the contract names in our offchain codebase should match the actual contract names. And the BLS contract no longer has the BLS prefix, its just called OperatorStateRetriever. Hence this PR.